releng/org.codechecker.eclipse.target/org.codechecker.eclipse.target.target

```diff
@@ -8,7 +8,7 @@
 <unit id="org.eclipse.swtbot.forms.feature.group" version="2.7.0.201806111355"/>
 <unit id="org.eclipse.swtbot.generator.feature.feature.group" version="2.7.0.201806111355"/>
 <unit id="org.eclipse.swtbot.ide.feature.group" version="2.7.0.201806111355"/>
-<repository location="http://download.eclipse.org/technology/swtbot/releases/latest"/>
+<repository location="http://download.eclipse.org/technology/swtbot/releases/2.7.0"/>
```

Comment:
SWTBot update site, changed in the meantime. Needed to reference the versioned repository.